lesson-4/css/style.css

```diff
@@ -1,95 +1,146 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('../fonts/roboto-medium.woff2') format('woff2');
+    font-style: normal;
+    font-weight: 500;
+}
+
+@font-face {
+    font-family: 'Roboto';
+    src: url("../fonts/roboto-regular.woff2") format('woff2');
+    font-style: normal;
+    font-weight: 400;
+}
+
+body {
+    margin: 0;
+}
+
+.what-do-we-offer {
+    height: 569px;
+}
 
 .title {
-    position: absolute;
-    width: 306px;
-    height: 47px;
-    left: 567px;
-    right: 567px;
-    top: 120px;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
     font-size: 40px;
     margin: 0;
+    display: flex;
+    justify-content: center;
+    padding-top: 120px;
 }
 
 hr {
-    position: absolute;
     width: 100px;
     height: 4px;
-    left: 670px;
-    top: 187px;
     border: 0;
-    margin: 0;
     background: #F5BD41;
 }
 
+#advantages {
+    width: 1233px;
+    height: 184px;
+    font-family: Roboto, sans-serif;
+    font-weight: 400;
+    font-size: 24px;
+    margin: auto;
+    display: flex;
+    justify-content: space-between;
+    padding: 80px 103px 114px 104px;
+}
+
 .container-support {
-    position: absolute;
     width: 296px;
     height: 176px;
-    left: 104px;
-    top: 271px;
-}
-
-.container-support img {
-    margin-left: 110px;
-    margin-right: 110px;
-    margin-bottom: 52px;
+    text-align: center;
 }
 
 .container-support p {
-    font-size: 24px;
-    margin: 0 78px 0 77px;
-    text-align: center;
+    margin-top: 49px;
 }
 
 .container-communication {
-    position: absolute;
     width: 296px;
     height: 178px;
-    left: 416px;
-    top: 271px;
-}
-
-.container-communication img {
-    margin: 8px 107px 56px 105px;
+    text-align: center;
 }
 
 .container-communication p {
-    font-size: 24px;
-    margin: 0 32px 0 32px;
-    text-align: center;
+    margin-top: 56px;
 }
 
 .container-booking {
-    position: absolute;
     width: 296px;
     height: 184px;
-    left: 728px;
-    top: 271px;
-}
-
-.container-booking img {
-    margin: 16px 98px 64px 98px;
+    text-align: center;
 }
 
 .container-booking p {
-    font-size: 24px;
-    margin: 0 80px 0 90px;
-    text-align: center;
+    margin-top: 72px;
 }
 
 .container-reviews {
-    position: absolute;
     width: 296px;
     height: 183px;
-    left: 1041px;
-    top: 271px;
+    text-align: center;
 }
 
-.container-reviews img {
-    margin: 4px 102px 60px 102px;
+.container-reviews p {
+    margin-top: 55px;
 }
 
-.container-reviews p {
-    font-size: 24px;
-    margin: 0 50px 7px 50px;
+/*footer*/
+
+footer {
+    color: #ffffff;
+    background-color: #10367C;
+    height: 630px;
+}
+
+.footer {
```

Comment:
don't use "footer" classname . this can be confusing since there is already a tag